esp/files/scripts/configmgr/configmgr.js

```diff
@@ -751,6 +751,18 @@ function initItemForRoxiePorts(item) {
   item.aclName = "";
   item.aclName_extra = "";
   item.aclName_ctrlType = 0;
+  item.protocol = "";
+  item.protocol_extra = "";
+  item.protocol_ctrlType = 0;
+  item.passphrase = "";
+  item.passphrase_extra = "";
+  item.passphrase_ctrlType = 0;
+  item.certificateFileName = "";
+  item.certificateFileName_extra = "";
+  item.certificateFileName_ctrlType = "";
```

Comment:
this element seems to be populated a little differently than the others, was this intentionally done this way?